examples/Advanced_Sounding_With_Complex_Layout.py

```diff
@@ -24,60 +26,47 @@
 ###########################################
 # Upper air data can easily be obtained using the siphon package,
 # but for this example we will use some of MetPy's sample data.
-
 col_names = ['pressure', 'height', 'temperature', 'dewpoint', 'direction', 'speed']
-
 df = pd.read_fwf(get_test_data('may4_sounding.txt', as_file_obj=False),
                  skiprows=5, usecols=[0, 1, 2, 3, 6, 7], names=col_names)
-
 # Drop any rows with all NaN values for T, Td, winds
 df = df.dropna(subset=('temperature', 'dewpoint', 'direction', 'speed'),
                how='all').reset_index(drop=True)
-
 ###########################################
 # We will pull the data out of the example dataset into
 # individual variables and assign units.
-
 p = df['pressure'].values * units.hPa
 z = df['height'].values * units.m
 T = df['temperature'].values * units.degC
 Td = df['dewpoint'].values * units.degC
 wind_speed = df['speed'].values * units.knots
 wind_dir = df['direction'].values * units.degrees
 u, v = mpcalc.wind_components(wind_speed, wind_dir)
-
 ###########################################
 # Now lets make a Skew-T Log-P diagram using some simply
 # MetPy functionality
-
 # Create a new figure. The dimensions here give a good aspect ratio
 fig = plt.figure(figsize=(9, 9))
 add_metpy_logo(fig, 430, 30, size='large')
```

Comment:
The logo is currently plotting over the data in the figure in the draft docs, so it would be best to move it elsewhere. Could go either lower right or lower left.